config/dosarray_config.sh

```diff
@@ -39,11 +56,9 @@ function dosarray_scp_from () {
   local FROM="$2"
   local TO="$3"
   scp -r -P <SSH_PORT> <USERNAME>@${HOST_NAME}.<FULLY_QUALIFIED_NAME>:${FROM} ${TO}
-
 }
 export -f dosarray_scp_from
 
-# FIXME update other scripts to use these:
```

Comment:
Was this line removed deliberately? The commit message doesn't specify.

Reply:
Yes it was, the other scripts are updated these variables appropriately so the FIXME is no longer needed. Will make sure to add such info in following commits